File: CS/15_Document/ChangePdfVersion.cs
using Spire.Pdf;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace ChangePdfVersion
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            // Open a PDF document
            PdfDocument doc = new PdfDocument();
            doc.LoadFromFile(@"..\..\..\..\..\..\Data\ChangePdfVersion.pdf");

            // Change the PDF version to Version 1.6
            doc.FileInfo.Version = PdfVersion.Version1_6;

            // Save the modified PDF document to a new file
            doc.SaveToFile("ChangePdfVersion_result.pdf");

            //Launch the Pdf file.
            PDFDocumentViewer("ChangePdfVersion_result.pdf");
        }

        private void PDFDocumentViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }
    }
}

File: VB/15_Document/ChangePdfVersion.vb
Imports Spire.Pdf

Namespace ChangePdfVersion
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			' Create a new PdfDocument object.
			Dim doc As New PdfDocument()

			' Load an existing PDF file from the specified path.
			doc.LoadFromFile("..\..\..\..\..\..\Data\ChangePdfVersion.pdf")

			' Set the PDF version of the loaded document to Version 1.6.
			doc.FileInfo.Version = PdfVersion.Version1_6

			' Save the modified document to a new file named "ChangePdfVersion_result.pdf".
			doc.SaveToFile("ChangePdfVersion_result.pdf")

			' Close the document.
			doc.Close()

			' Launch the Pdf file.
			PDFDocumentViewer("ChangePdfVersion_result.pdf")
		End Sub

		Private Sub PDFDocumentViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub
	End Class
End Namespace
